### ISSUE TITLE
Tooltips don't disappear after click on tooltipped label

### ISSUE THREAD

**Issue by AlbertoDrusiani** [Dogfalo/materialize#5824]
Tooltip works properly unless when clicking on the label.
Input inside the label has a click handler without any return false or stop propagation or anything involved in tooltip or label modification.

THE CODE
```
<label class="tooltipped" data-position="bottom" data-tooltip="Inside code debug" for="debug">
    <input type="checkbox" id="debug" class="logLegendCheck" checked/>
    <span>DEBUG</span>
</label>
```

When checkbox is clicked, the tooltip doesn't disappear.
Materialize 1.0.0-beta

**Comment by acburst** [comment]
Copying that code worked for me on my test page. Can you create a codepen that reproduces the problem?

**Comment by AlbertoDrusiani** [comment]
https://codepen.io/anon/pen/ZoELxw

**Comment by Mzeus** [comment]
I have the same problem with some links that open modals while using tooltip in v1.0.0-beta.
After you click to open the modal sometimes the tooltip doesnt desappear. I tried to reproduce this steps, but cant figure out the patern. Some times it works sometimes doesnt.
I didn't have this problem using v1.0.2-alpha.

**Comment by Dogfalo** [comment]
Fixed in 8399a44632a4996b89fcbad92dc6b2e1ce5266e1
